setup.py

```diff
@@ -6,11 +6,9 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-requirements = read("requirements.txt").split()
```

Comment:
Should we also remove this file if it's not used?

Reply:
The file is still used for the `api`.

Note: Whether we should keep a separated api is an ongoing reflexion (#47).